### DISCUSSION TITLE
self-hosting/

### DISCUSSION THREAD

**Discussion by giscus** [discord-tickets/bot#560] [Documentation]
# self-hosting/

The most popular open-source ticket management bot for Discord.

https://discordtickets.app/self-hosting/

**Comment by Adam9000385** [comment]
100

**Reply by gonsalvespatrick502-dot** [reply]
Olá seja bem vindo

**Comment by hdgcbshchduc** [comment]
ولايه سيتي

**Comment by hdgcbshchduc** [comment]
DR7

**Reply by hdgcbshchduc** [reply]
وانر- فادرن-ادري عليه-مبرمج -
